tcf_website/templates/login/login_modal.html

```diff
@@ -0,0 +1,27 @@
+<!-- Login modal (imported in index.html) -->
+{% load static %}
+<div class="modal fade" id="loginModal" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Sign In</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+          <div class="d-flex justify-content-center">
```

Comment:
Would this be a place to use your `logo-tagline`?

Reply:
I tried but it looks kinda bad in the modal on small screens rip

Reply:
Ok maybe something to look into later
We need to put that refactor to use 😄